systems/notification/mailer/test/integration/mailer_test.go

```diff
@@ -47,8 +47,8 @@ func TestSendEmailAndGetEmail(t *testing.T) {
 	t.Run("SendEmail", func(t *testing.T) {
```

Comment:
Also, integration tests in  this file github.com/ukama/ukama/systems/notification/mailer/test/integration/mailer_test.go are failing. Please, make sure you fix those.

Reply:
test fixed